addons/mail/static/src/models/notification_group/notification_group.js

```diff
@@ -67,6 +67,14 @@ function factory(dependencies) {
             return `${this.modelName}_${data.id}`;
         }
 
+        _computeDate() {
+            return moment.max(this.notifications.map((notification) => notification.message.date));
+        }
+
+        _computeSequence() {
+            return -Math.max(...this.notifications.map((notification) => notification.id));
+        }
```

Comment:
It was supposed to use `notification.message.id` no?

Reply:
Yes, it's